sulong/projects/com.oracle.truffle.llvm.runtime/src/com/oracle/truffle/llvm/runtime/LLVMLanguage.java

```diff
@@ -56,7 +56,8 @@
     public static final String MAIN_ARGS_KEY = "Sulong Main Args";
     public static final String PARSE_ONLY_KEY = "Parse only";
 
-    public static final String NAME = "llvm";
+    public static final String ID = "llvm";
+    public static final String NAME = "LLVM";
```

Comment:
You probably also need to change the usages of `LLVMLanguage.NAME` to use `LLVMLanguage.ID` instead (I expect at least some usages in the tests). That will probably fix the travis failures.

Reply:
Thanks, should be fixed via https://github.com/oracle/graal/pull/820/commits/9cb798a89be5c9f83274aa51e3ff75678aa7d161.

Reply:
Thanks. I'll merge it.